website/docusaurus.config.js

```diff
@@ -109,24 +113,6 @@ var siteSettings = {
           position: 'left',
           activeBasePath: 'reference'
         },
-        {
-          to: '/dbt-cli/cli-overview',
-          label: 'dbt CLI',
-          position: 'left',
-          activeBasePath: 'dbt-cli'
-        },
-        {
-          to: '/docs/dbt-cloud/cloud-overview',
-          label: 'dbt Cloud',
-          position: 'left',
-          activeBasePath: 'docs/dbt-cloud'
-        },
-        {
-          to: '/guides/getting-started',
-          label: 'Guides',
-          position: 'left',
-          activeBasePath: 'guides'
-        },
```

Comment:
Yea @JKarlavige We should probably hold off moving [dbt-cloud and dbt-cli ](https://github.com/dbt-labs/docs.getdbt.com/pull/1990/files#diff-bc4913d026ae51bef2bc8b7a1c040fdf52b999d99e6fa40d87b379f98a60e0eaL112-L123) until the content is moved. But "Guides" can definitely go under "Learn"

Reply:
if we wanna move "Guides" under "Learn" (fka "Courses") in this PR, does it make sense to move "Contributing" from the left sidebar under "Community" as well to get that part done.

or is it better to leave all nav changes in another PR? scope-wise, this might make sense given the video and figma in the Asana ticket.

Reply:
Hi all. I've added these items back so this should be fixed.